snapshots/overlay/overlayutils/check.go

```diff
@@ -88,6 +94,21 @@ func Supported(root string) error {
 	return SupportsMultipleLowerDir(root)
 }
 
+// IsPathOnTmpfs returns whether the path is on a tmpfs or not.
+//
+// It uses statfs to check if the fs type is TMPFS_MAGIC (0x01021994)
+// see https://man7.org/linux/man-pages/man2/statfs.2.html
+func IsPathOnTmpfs(d string) bool {
+	stat := syscall.Statfs_t{}
+	err := syscall.Statfs(d, &stat)
+	if err != nil {
+		log.L.WithError(err).Warnf("Could not retrieve statfs for %v", d)
+		return false
```

Comment:
I want to return this error, but this is coming from @AkihiroSuda's comment.

@AkihiroSuda - The errors on https://man7.org/linux/man-pages/man2/statfs.2.html#ERRORS seem pretty rare. Other operations would fail if statfs fails. So is it really make sense to log and essentially ignore the error?

Reply:
Maybe not pretty rare on fuse